src/components/NcHighlight/NcHighlight.vue

```diff
@@ -98,7 +111,7 @@ export default {
 					})
 				}
 				return validRanges
-			}, [])
+			}, [] as StringRange[])
```

Comment:
Maybe, better use this?
```
ranges = ranges.reduce<StringRange[]>()
```

Reply:
No strong preference. So your proposal looks cleaner, will adjust it :)